chess/ChessGameModel.swift

```diff
@@ -134,6 +137,14 @@ extension ChessGameModel {
     }
 
     private func handle(_ message: String) {
+        guard isValidLAN(message) || isValidColor(message) else {
+            self.logger.error("Received invalid message in Model: \(message)")
```

Comment:
```suggestion
            self.logger.error("Received invalid message: \(message)")
```
not sure why we need to specify model here so suggest to omit this word